### DISCUSSION TITLE
Why is Scan rejected

### DISCUSSION THREAD

**Discussion by tomberlin55** [rspamd/rspamd#4253] [General]
I have a network scanner that sends the scan via e-mail. sometimes the scan is rejected, sometimes not.

<html>
<body>

1661611988.798176.334@BR5CF370BEB457 | 80.237.133.45 | scanner@mds4u.de | ballt@mds4u.de | Scan to E-mail Server Job | reject | 0.00 / 20 | 5.09M | 0.050 | 27.8.2022, 15:53:28 | unknown
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
[Envelope To] To/Cc/Bccballt@mds4u.deSymbolsSort by:MagnitudeValueNameCONTENT_BLACKLISTED (0)ASN (0) [asn:34011, ipnet:80.237.132.0/23, country:DE] | ballt@mds4u.de | CONTENT_BLACKLISTED (0)ASN (0) [asn:34011, ipnet:80.237.132.0/23, country:DE]
ballt@mds4u.de
CONTENT_BLACKLISTED (0)ASN (0) [asn:34011, ipnet:80.237.132.0/23, country:DE]

</body>
</html>
Thats the message in the log file

Why do they get rejected wir a value of 0.00/20?

Would I have to put the sending mail adress scanner@mds4u.de in whitelist and thats all?
Thomas

**Comment by moisseev** [comment]
> Thats the message in the log file 

More like a copy/paste from WebUI History table.

> Why do they get rejected wir a value of 0.00/20?

I guess you have forced reject action somewhere. Probably for CONTENT_BLACKLISTED symbol.
You should check **all** lines related to the message in **the log file**.

**Reply by tomberlin55** [reply]
The log file is empty, probably because the server restartet...
I have several BLACKLISTED settings, but non regard to the scanner@mds4u.de address.
The strange thing is, that all other mail came through only one was rejected.
Maybe restarting the server helped.
